sources/system/file-system/file-system.dylan

```diff
@@ -516,3 +516,19 @@ define sideways method list-locator
      locator);
   locators
 end method list-locator;
+
+///
+
+define generic symbolic-link
```

Comment:
This could use a test.  See also `test-file-exists?`.  You can use `test-temp-directory`.  Looks like I was lazy and didn't write a test using `run-application("ln -s ...)`.